### ISSUE TITLE
coredns notReady when  reload

### ISSUE THREAD

**Issue by wwgfhf** [coredns/coredns#3387]
I use coredns in kubernetes, when i change the cm , and reload worked.

But then the coredns notReady

```
[INFO] Reloading
2019-10-21T01:36:17.116Z [INFO] plugin/reload: Running configuration MD5 = b9d55fc86b311e1d1a0507440727efd2
[INFO] Reloading complete
```
```
Readiness probe failed: Get http://10.151.32.7:8181/ready: dial tcp 10.151.32.7:8181: connect: connection refused
```
This is my corefile, and the version of coredns is 1.5.0
```
.:53 {
        errors
        health
        ready
        kubernetes cluster.local in-addr.arpa ip6.arpa {
          pods insecure
          upstream /etc/resolv.conf
          fallthrough in-addr.arpa ip6.arpa
        }
        prometheus :9153
        forward . /etc/resolv.conf {
          prefer_udp
        }
        cache 30
        loop
        reload
        loadbalance
    }
```

**Comment by chrisohaver** [comment]
IIRC, this was a bug in 1.5.0, and fixed in a later version.

**Comment by chrisohaver** [comment]
Yeah - fixed in 1.5.1: https://github.com/coredns/coredns/blob/master/notes/coredns-1.5.1.md
